frontend/packages/console-shared/src/types/node.ts

```diff
@@ -0,0 +1,13 @@
+export type NodeCondition = {
+  lastHeartbeatTime: string;
+  lastTransitionTime: string;
+  message: string;
+  reason: string;
+  status: string;
+  type: string;
+};
+
+export type NodeAddress = {
+  type: string;
```

Comment:
Side note, for enumerated (known) values, we could also use a union of string literals, like `'ip4' | 'ip6'` which is technically more specific than just `string`.

Reply:
It seems that the type can vary. I'll keep this in mind and update this subsequently when I know more.